help/destinations/catalog/crm/hubspot.md

```diff
@@ -0,0 +1,188 @@
+---
+title: HubSpot connection
+description: The HubSpot destination allow you to manage contact records in your HubSpot account.
+last-substantial-update: 2023-08-15
+---
+# [!DNL HubSpot] connection
+
+[[!DNL HubSpot]](https://www.hubspot.com) is a CRM platform with all the software, integrations, and resources you need to connect marketing, sales, content management, and customer service. It allows you to connect your data, teams, and customers on one CRM platform.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL HubSpot] Contacts API](https://developers.hubspot.com/docs/api/crm/contacts), to update contacts within [!DNL HubSpot] from an existing [!DNL HubSpot] audience after activating them within a new segment.
```

Comment:
"from an existing [!DNL HubSpot] audience" -> shouldn't this be "from an existing Experience Platform audience"?